app/locales/gj/exportscreen.json

```diff
@@ -0,0 +1,8 @@
+{
+    "export_para_1":"તમે નીચે શેર કરો બટનનો ઉપયોગ કરીને કોઈપણ સાથે તમારી સ્થાન પગેરું શેર કરી શકો છો. એકવાર તમે બટન દબાવો તે તમને પૂછશે કે તમે કોની સાથે અને કેવી રીતે તેને શેર કરવા માંગો છો.",
+    "export_para_2":"સ્થાન ઇતિહાસ ફક્ત સ્થાનો અને સમયની સૂચિ તરીકે બતાવેલ છે, તેમાં કોઈ વ્યક્તિગત માહિતી શામેલ નથી",
+    "share":"શેર કરો",
+    "data_hint":"યાદીમાં માહિતી છે"
+}
```

Comment:
new file has more key-value pairs, needs to be updated with translation for those (see locales/en/exportscreen.json for more)

Reply:
done

Reply:
excellent!